module1.js

```diff
@@ -51,10 +61,10 @@ function question3(number) {
   For example, if the input array is: [1,2,3,4,5,6]
   the output would be: [2,4,6]
 */
-
-function question4(array) {
-  // TODO:
-}
+var arr = [4,5,7,8,14,45,76];
```

Comment:
It's the job of the _test_ to supply values when calling your function, you don't have to supply your own values in the implementation. Again this is about reusability, we don't want the test to care about specific values.